building/tracks/syllabus/next-exercises.md

```diff
@@ -0,0 +1,27 @@
+# Syllabus - Next Exercises
```

Comment:
One additional thing we might want to add on this page is to make the cyclic dependency issue a bit more tangible as it happens most often in this part of the tree and it is not fixable by "writing some other exercise later" as described at the end of the document.
E.g. we could give an example that loops can only use arrays if they have been introduced before. This in turn means that the exercise for arrays does not require the student to use a loop.

Probably the more general point here is to plan ahead a little as the situation above can easily happen even with just 2 people of the core team writing the first couple of exercises. I know we want to make this sound as easy as possible but there are hard to fix corners that you can get into.

Reply:
Yeah, that's a good call. I'll elaborate a bit here as well. Thank you!